src/core/custom/msgs/MsgAminoCustom.ts

```diff
@@ -0,0 +1,80 @@
+import { JSONSerializable } from '../../../util/json';
+
+/**
+ * A delegator can submit this message to send more alliance assets
+ * to be staked through the alliance module in a validator.
+ */
+export class MsgAminoCustom extends JSONSerializable<
+  MsgAminoCustom.Amino,
+  MsgAminoCustom.Data,
+  {}
+> {
+  public aminoOnly = true;
```

Comment:
If you delete the amunoOnly you may be able to skip casting the msgs here https://github.com/terra-money/feather.js/pull/11/files#diff-7cea133731caa87c1ed3eebf44f96a14be2cb2c8421663e6c100a4617b041123L168 since anyway is not used

```suggestion
```